server/config/options.go

```diff
@@ -388,7 +388,6 @@ var DefaultConfig = Config{
 		Compression:       false,
 		IgnoreExtraFields: false,
 		LogFilter:         false,
-		AuthKey:           "ts_test_key",
```

Comment:
Will this break tests?

Reply:
No. It was added accidentally in previous commit.